_data/banking.yml

```diff
@@ -163,6 +163,14 @@ websites:
       phone: Yes
       doc: https://www.mybankofinternet.com/tob/live/usp-core/static/help.html#login_security
 
+    - name: Caisse D'Eparnge
```

Comment:
On their Facebook and Twitter page they've named themselves Caisse d'Epargne with a lowercase d. I don't speak French so I won't try to request any changes here but if it is a typo here you might want to change that. 😉 
If not then this PR is good to go.